apps/mobile/src/queries/balance/account-balance.query.ts

```diff
@@ -23,10 +24,10 @@ interface AccountBalance {
   totalBalance: FetchState<Money>;
 }
 
-const zeroMoneyUsd = createMoney(0, 'USD');
-
 export function useAccountBalance(accountId: AccountId): AccountBalance {
   const { fingerprint, accountIndex } = accountId;
+  const { fiatCurrencyPreference } = useSettings();
+  const zeroMoneyFiat = createMoney(0, fiatCurrencyPreference);
```

Comment:
nit: outside the `useAccountBalance` this would only instantiate once, thought I doubt this particular call would have any measurable impact.

Reply:
My bad, didn't realize that. Not sure if it's worth the trouble unless there's measurable impact.